ui/pages/routes/routes.container.js

```diff
@@ -64,6 +72,23 @@ function mapStateToProps(state) {
       !connectedAccounts.find((address) => address === account.address),
   );
 
+  const networkToAutomaticallySwitchTo =
+    getNetworkToAutomaticallySwitchTo(state);
+  const allNetworks = getAllNetworks(state);
+  const switchedNetworkDetailsObj = getSwitchedNetworkDetails(state);
```

Comment:
Nit: This could all be incorporated in the `getSwitchedNetworkDetails` selector. It would be more easily tested that way as well.

Reply:
Done!